src/contract-interact.ts

```diff
@@ -52,20 +52,24 @@ export async function interactWrite(
  * @param target        if needed to send AR to an address, this is the target.
  * @param winstonQty    amount of winston to send to the target, if needed.
  * @param myState       a locally-generated state variable
+ * @param fromParam     The from address of the transaction
+ * @param contractInfoParam The loaded contract
  */
 export async function interactWriteDryRun(
   arweave: Arweave,
   wallet: JWKInterface | 'use_wallet',
   contractId: string,
   input: any,
+  myState: any={},
```

Comment:
Please add any extra params to functions at the end of it, with a default, to prevent current users using the code from having breaking changes on updates.